marker_map:           aruco-26-70x70-5
enclosed_marker:      false
detection_mode:       0
marker_frame:         b_bot_ar_marker
reference_frame:      workspace_center
camera_parent_frame:  workspace_center
robot_name:           b_bot
robot_effector_frame: b_bot_gripper_base_link
initpose:             [0.10,  0.10, 0.35, 0, 90, 0]
keyposes:
  - [0.10,  0.15, 0.16, 0, 25, 0]
  - [0.10,  0.05, 0.24, 0, 25, 0]
  - [0.10, -0.05, 0.32, 0, 25, 0]
  - [0.10, -0.15, 0.24, 0, 25, 0]
  - [0.10, -0.25, 0.16, 0, 25, 0]

  - [-0.05, -0.25, 0.32, 0, 25, 0]
  - [-0.05, -0.15, 0.24, 0, 25, 0]
  - [-0.05, -0.05, 0.16, 0, 25, 0]
  - [-0.05,  0.05, 0.21, 0, 25, 0]
  - [-0.05,  0.15, 0.26, 0, 25, 0]
